app/views/user_registrations/link_github.html.slim

```diff
@@ -0,0 +1,19 @@
+.page-header
+  h3.text-center.green-text Welcome on board! Let's get started
+  
+  p.text-center During the program, you will need a Github repository and account to share your awesome code with your mentor.
+
+  p.text-center Don't have a GitHub account and repo? Check this guide on how to create one! 
+
```

Comment:
@alma-castant yes this page is no more. That is why I was asking to have atleast git tutorail link on dashboard and ask them that their commit will be observed throughout out the program. Just a suggestion. Also if u remove the git page and don't enforce them, than what if they wont make it till end of the program ? And just push it on last day. Then there is no point keeping track based on commits. Like for example in my case, since i didn't know git, i keep postponing till end of the time. Until its mandatory. 

So ill suggest to keep the page, and give them an option to move forward even if they don't fill the git link. Also add git link tutorial on dashboard, and add somewhere, that git commits will be observed for project performance, so that will motivate them to create account on git early :)